src/main/java/org/truffleruby/options/CommandLineParser.java

```diff
@@ -557,8 +545,12 @@ private String grabOptionalValue() {
         return null;
     }
 
+    private CommandLineException notImplemented(String option) {
+        return new CommandLineException(String.format("the %s option is not implemented", option));
+    }
+
     public static void printHelp(PrintStream out) {
-        out.println("Usage: truffleruby [switches] [--] [programfile] [arguments]");
+        out.println("Usage: ruby [switches] [--] [programfile] [arguments]");
```

Comment:
Should the usage really print `ruby`?
I would find it very confusing if there is no way to discern if I just invoked MRI or not.
Startup could be one way to differentiate but that doesn't work on SVM :D
I think `truffleruby` is clearer.
Alternatively something based on the value of Xlauncher but that seems too long for here.